### ISSUE TITLE
Connector Housekeeping

### ISSUE THREAD

**Issue by BentsiLeviav** [ClickHouse/power-bi-clickhouse#30]
There are two things we would like to change, that would provide a better user experience:
1. Link the learn more part to the connector's official docs instead of the ClickHouse home page
![image](https://github.com/user-attachments/assets/537fc4f1-f63d-40ae-9a3e-566d486964b8)

2. When throwing an exception on ODBC absence, provide a link to the ODBC installation docs
![image](https://github.com/user-attachments/assets/5c054155-c9e4-49d4-98b2-3e8bc6896a5d)

**Comment by BentsiLeviav** [comment]
Would also want to update the docs to include new screenshots (new versions of PowerBI and ClickHouse Cloud)

**Comment by BentsiLeviav** [comment]
Would also want to turn off the beta flag

**Comment by BentsiLeviav** [comment]
We also need to update the [ClickHouse docs](https://clickhouse.com/docs/en/integrations/data-visualization) regarding our support for PowerBI service.

**Comment by laeg** [comment]
We should also update the label in ODBC connector setup screen to be `Host` instead of `Server` to align with our documentation/naming convention

![image](https://github.com/user-attachments/assets/326670f1-19de-413e-9648-eb05bf7fcc2f)

**Comment by BentsiLeviav** [comment]
* The documentation was updated at #ClickHouse/clickhouse-docs/pull/2734
* We couldn't update CH cloud screenshots, as we import them as a snippet (and many other pages use it). I opened an issue in the docs repo #ClickHouse/clickhouse-docs/issues/2733
* A link to the ODBC installation guide link is provided when the connector throws an exception about missing driver
* The "learn more" link was changed to the [official docs](https://clickhouse.com/docs/en/integrations/powerbi)
* The Beta flag was removed
* The field name was changed from Server to Host (due to the fact that the parameter name in ODBC is Server, I kept the code as is and only changed the UI).

All these changes are in [changes-for-certification-process](https://github.com/ClickHouse/power-bi-clickhouse/tree/changes-for-certification-process)

**Comment by laeg** [comment]
@BentsiLeviav i noticed [the images](https://github.com/ClickHouse/clickhouse-docs/pull/2734/commits/a683231693d0dd0ae612fbebeb03238f64471887) still show Beta? will we need to another docs update later?

**Comment by BentsiLeviav** [comment]
@laeg That's right, it is either having a screenshot with a beta flag or with a Custom indication like the following:
![image](https://github.com/user-attachments/assets/c4786ec3-ebce-417f-b74d-8ea95df4f956)

Once the MS team publishes the new version, we will have a clean screenshot to add. 
I'll reopen this issue to keep this in mind.

**Comment by BentsiLeviav** [comment]
We changed the screenshot in the docs to the latest version (without the `beta` flag)
https://github.com/ClickHouse/clickhouse-docs/pull/3229
closing this.
